src/main/java/graphql/language/OperationDefinition.java

```diff
@@ -13,7 +13,7 @@
 import static graphql.language.NodeChildrenContainer.newNodeChildrenContainer;
 
 @PublicApi
-public class OperationDefinition extends AbstractNode<OperationDefinition> implements Definition<OperationDefinition>, SelectionSetContainer<OperationDefinition> {
+public class OperationDefinition extends AbstractNode<OperationDefinition> implements Definition<OperationDefinition>, SelectionSetContainer<OperationDefinition>, DirectivesContainer<OperationDefinition>, NamedNode<OperationDefinition> {
 
```

Comment:
OperationDefintion doesn't always has a name, right?

Reply:
true - we could make it "" OR note makes it a NamedNode

Reply:
I made it not a named node